ci/launch_cluster.sh

```diff
@@ -50,5 +47,8 @@ if ! ./dcos-launch wait; then
   exit 3
 fi
 
+# Extract SSH key
+jq -r .ssh_private_key cluster_info.json > "$CLI_TEST_SSH_KEY"
```

Comment:
I don't see the CLI_TEST_SSH_KEY used anywhere in this script so what is the point of this code?

Reply:
oh, I get it now. Thanks!